portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/Cookies.testcase

```diff
@@ -9,13 +9,91 @@ definition {
 	setUp {
 		task ("Setup: Setup portal instance") {
 			TestCase.setUpPortalInstance();
+
+			User.firstLoginPG();
+		}
+	}
+
+	@description = "Combined AC-1, AC-2, AC-3, AC-4"
+	test AC1 {
+		property portal.upstream = "gabor";
+		property test.name.skip.portal.instance = "Cookies#AC1";
+
+		task ("Given: a site configured to show the Banner") {
+			CookieBannerAdmin.configuredToShowBanner();
+		}
+
+		task ("When: the End User first visits the site") {
+			CookieHelper.siteVisit();
+		}
+
+		task ("Then: they see the Banner text as the Description is configured for the Banner in the site administration.") {
+			CookieBanner.verifyCustomTextsArePresent();
+		}
+
+		task ("And: they see the 'Decline All' button on the right side of the Banner, next to the 'Accept All' button.") {
```

Comment:
Not 100% sure, but KC mentioned that don't use **And** under(or after) **Then** otherwise separate them into two cases.
e.g. https://gist.github.com/reizero00/f6b9af59412b3473a89d2017c7d0d570

I'm OK with the current structure, it maybe better that you double check with KC about the details of the usage of Then-And.